sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerContext.java

```diff
@@ -49,6 +49,7 @@
   public static final String CTX_SQL_QUERY_ID = "sqlQueryId";
   public static final String CTX_SQL_CURRENT_TIMESTAMP = "sqlCurrentTimestamp";
   public static final String CTX_SQL_TIME_ZONE = "sqlTimeZone";
+  public static final String CTX_SQL_OUTER_LIMIT = "sqlOuterLimit";
```

Comment:
should this be documented somewhere or just leave it hidden since I guess is mostly to be friendly to web console?

Reply:
I think it would be better to document internal parameters somewhere rather than depending on human memory. But I think we can do in a follow-up pr.

Reply:
I think this should be in the docs, it is a super useful parameter IMO even outside the web console